src/components/shareable-url-modal/shareable-url-metadata.js

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react';
+import { isRunningLocally } from '../../utils';
+
+const ShareableUrlMetadata = () => {
+  const [metadata, setMetadata] = useState(null);
+
+  useEffect(() => {
+    if (isRunningLocally()) {
```

Comment:
can we have this check when the component is mounted like when it is used in the page instead of checking it here ?

Reply:
Updated.